### PR TITLE
Update user guide

### LINKED ISSUE #26: Update UserGuide.md
Update user guide for assigned features

### PR DESCRIPTION
**Pull request by VaishakAnand**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -172,7 +177,7 @@ Action | Format, Examples
 **Add** | `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​` <br> e.g., `add n/James Ho p/22224444 e/jamesho@example.com a/123, Clementi Rd, 1234665 t/friend t/colleague`
 **Clear** | `clear`
 **Delete** | `delete INDEX`<br> e.g., `delete 3`
-**Edit** | `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`<br> e.g.,`edit 2 n/James Lee e/jameslee@example.com`
+**Edit** | `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [n/NAME] [p/PHONE] [v/CLASS_VENUE] [s/SCHOOL] [sb/SUBJECT] [y/YEAR] [t/CLASS_TIME] [a/ADDITIONAL_DETAILS]`<br> e.g.,`edit 1 n/Alex p/99999999 s/AJC`
```
Comment (AlexCQY) [inline docs/UserGuide.md]: Might want to remove the email part as our student model doesnt have email.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -87,22 +87,27 @@ Shows a list of all persons in the address book.
...
 * Existing values will be updated to the input values.
-* When editing tags, the existing tags of the person will be removed i.e adding of tags is not cumulative.
-* You can remove all the person’s tags by typing `t/` without
-    specifying any tags after it.
 
 Examples:
-*  `edit 1 p/91234567 e/johndoe@example.com` Edits the phone number and email address of the 1st person to be `91234567` and `johndoe@example.com` respectively.
-*  `edit 2 n/Betsy Crower t/` Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.
+*  `edit 1 n/Alex p/99999999 s/Meridian Junior College` Edits the name, phone number and school of the 1st person to be `Alex`, `99999999` and `Meridian Junior College` respectively.
+*  `edit 3 e/amongus@isgood.com sb/Mathematics v/Blk 33 West Coast Rd #21-214 t/1 1430-1630` Edits the email address, subject, venue and time of the third student to be `amongus@isgood.com`, `Mathematics`, `Blk 33 West Coast Rd #21-214` and `1 1430-1630` respectively.
```
Comment (AlexCQY) [inline docs/UserGuide.md]: Might want to remove the "e/amongus@isgood.com" part and also the "email address"

**Review by hogantan — APPROVED** [review]
Other than the part that Alex point out about the email, LGTM

File: docs/UserGuide.md (hunk 1 above)
Comment (VaishakAnand) [reply docs/UserGuide.md]: Oops! Thank you, i've fixed it!

File: docs/UserGuide.md (hunk 2 above)
Comment (VaishakAnand) [reply docs/UserGuide.md]: Fixed, thanks!

**Review by csiongn — APPROVED** [review]

**Review by ong-yinggao98 — APPROVED** [review]
Nice

File: docs/UserGuide.md (hunk 1 above)
Comment (AlexCQY) [reply docs/UserGuide.md]: nice

File: docs/UserGuide.md (hunk 2 above)
Comment (AlexCQY) [reply docs/UserGuide.md]: nice

**Review by AlexCQY — APPROVED** [review]
LGTM

**Pull request merged**